### DISCUSSION TITLE
paragraph end is the same as previous paragraph end (and also earlier than the last sentence end)

### DISCUSSION THREAD

**Discussion by benwoodward** [deepgram/community#210] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

Here are the last two paragraphs in the JSON returned from the API:

```json
  {
    "sentences": [
      {
        "text": "한국에서 부모님과 만난 것도 이런 놀라운 기회였잖아.",
        "start": 124.7,
        "end": 128.5
      },
      {
        "text": "그들은 이렇게 멋진 경험을 우리에게 전해줬고, 이제 우리에게도 그 누군가와 같은 추억을 만들 수 있는 기회가 왔어.",
        "start": 129.88,
        "end": 139.26
      }
    ],
    "speaker": 0,
    "num_words": 24,
    "start": 124.7,
    "end": 139.26
  },
  {
    "sentences": [
      {
        "text": "두 사람은 타임캡슐에 담긴 그들의 추억들을 흐뭇하게 감상하며, 그 안에 한국에서 경험한 소중한 추억들이 남아있기를 소망했어요",
        "start": 139.54,
        "end": 150.56
      }
    ],
    "speaker": 1,
    "num_words": 16,
    "start": 139.54,
    "end": 139.26
  }
]
```

Notice that the `"end"` timestamp for the last paragraph (139.26):

- comes before the `end` timestamp of the last sentence in the paragraph (150.56)
- is the same as the previous paragraph's `end` timestamp

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

https://api.deepgram.com/v1/listen?model=whisper-large&punctuate=true&language=ko&diarize=true&smart_format=true&numerals=true&paragraphs=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

dd19b143-1765-4f34-a535-d78f829646a9

### If possible, please attach your code or paste it into the text box.

```js
  const fileSource = { url: mp3FileUrl }

  const deepgramResponse = await deepgram.transcription.preRecorded(fileSource, {
    model: 'whisper-large',
    punctuate: true,
    language: 'ko',
    diarize: true,
    smart_format: true,
    numerals: true,
    paragraphs: true,
  })
```

### If possible, please attach an example audio file to reproduce the issue.

[time+capsule.mp3.zip](https://github.com/deepgram/community/files/11740385/time%2Bcapsule.mp3.zip)

**Comment by atacnic** [comment]
Hi!  I am seeing the same issue did you have any luck fixing it?

**Comment by jkroll-deepgram** [comment]
@benwoodward @atacnic Thank you for sharing this issue. We have a resolution that will be released soon. The last paragraph's `end` timestamp will be the same as the last word's `end` timestamp. I will let you know once it's released.

**Reply by benwoodward** [reply]
oh great. glad you were able to reproduce it.

**Comment by jkroll-deepgram** [comment, marked as answer]
This issue has been resolved! Thanks again for raising it.
